static/dev/archive/get.html

```diff
@@ -49,7 +49,7 @@
 	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.7/jquery-ui.min.js"></script>
 	<script type="text/javascript">
 	
-	books = ["Amos", "Chronicles 1", "Chronicles 2", "Daniel", "Deuteronomy", "Ecclesiastes", "Esther", "Exodus", "Ezekiel", "Ezra Nehemiah", "Genesis", "Habakkuk", "Haggai", "Hosea", "Isaiah", "Jeremiah", "Job", "Joel", "Jonah", "Joshua", "Judges", "Kings 1", "Kings 2", "Lamenstaions", "Leviticus", "Malachi", "Nahum", "Numbers", "Obadiah", "Proverbs", "Psalms", "Ruth", "Samuel 1", "Samuel 2", "Song of Songs", "Zechariah", "Zephaniah"]
+	books = ["Amos", "Chronicles 1", "Chronicles 2", "Daniel", "Deuteronomy", "Ecclesiastes", "Esther", "Exodus", "Ezekiel", "Ezra Nehemiah", "Genesis", "Habakkuk", "Haggai", "Hosea", "Isaiah", "Jeremiah", "Job", "Joel", "Jonah", "Joshua", "Judges", "Kings 1", "Kings 2", "Lamentations", "Leviticus", "Malachi", "Nahum", "Numbers", "Obadiah", "Proverbs", "Psalms", "Ruth", "Samuel 1", "Samuel 2", "Song of Songs", "Zechariah", "Zephaniah"]
```

Comment:
This is the odd file I highlighted initially, I normally don't fix misspellings, but this seemed egregious, and perhaps the file doesn't need to live...